orderer/consensus/etcdraft/chain.go

```diff
@@ -394,6 +396,37 @@ func (c *Chain) Order(env *common.Envelope, configSeq uint64) error {
 // Configure submits config type transactions for ordering.
 func (c *Chain) Configure(env *common.Envelope, configSeq uint64) error {
 	c.Metrics.ConfigProposalsReceived.Add(1)
+
+	payload, err := protoutil.UnmarshalPayload(env.Payload)
+	if err != nil {
+		return errors.Wrap(err, "failed to extract payload from config envelope")
+	}
+	// get config update
+	configUpdate, err := configtx.UnmarshalConfigUpdateFromPayload(payload)
+	if err != nil {
+		return errors.Wrap(err, "could not read config update")
+	}
+
+	configMeta, err := MetadataFromConfigUpdate(configUpdate)
+	if err != nil {
+		return errors.Wrap(err, "could not read config metadata")
+	}
+
+	membershipUpdates, err := ComputeMembershipChanges(c.opts.BlockMetadata, c.opts.Consenters, configMeta.Consenters)
+	if err != nil {
+		c.logger.Panicf("illegal configuration change detected: %s", err)
+	}
+
+	if membershipUpdates.Rotated() {
+		lead := atomic.LoadUint64(&c.lastKnownLeader)
+		fmt.Printf("My raftID: %d Leader raftID: %d rotatedNode raftID: %d\n", c.raftID, lead, membershipUpdates.RotatedNode)
+		if membershipUpdates.RotatedNode == lead {
+			c.logger.Infof("Certificate of Raft leader is being rotated, attempt leader transfer before reconfiguring communication")
+			debug.PrintStack()
+			c.Node.abdicateLeader(lead)
```

Comment:
1. You also need to forward the request to the next leader. 
2. You need to abdicate only if the removed node is you, and you're the leader.

Reply:
> You need to abdicate only if the removed node is you, and you're the leader.

Updated the condition:
`if membershipUpdates.RotatedNode == lead && lead == c.raftID {`

Reply:
> You also need to forward the request to the next leader.

It is taken care already.